lib/PreviewProvider.js

```diff
@@ -26,12 +25,9 @@ const URL_FILTERS = [
 Cu.importGlobalProperties(["fetch"]);
 Cu.importGlobalProperties(["URL"]);
 Cu.import("resource://gre/modules/Task.jsm");
-Cu.import("resource://gre/modules/Services.jsm");
 
 const DEFAULT_OPTIONS = {
-  cacheCleanupPeriod: 86400000, // a cache clearing job runs at most once every 24 hours
-  cacheRefreshAge: 259200000, // refresh a link every 3 days
-  cacheTTL: 2592000000, // cached items expire if they haven't been accessed in 30 days
+  metadataTTL: 259200000,
```

Comment:
Just a personal nit, if you can represent times in ms like this as:

24 * 60 * 60 * 1000

it makes it easier to understand how long it is